pyproject.toml

```
@@ -66,7 +65,7 @@ version-file = "aiven/client/version.py"
 
 [tool.black]
 line-length = 125
-target-version = ['py37', 'py38', 'py39']  # 'py310' is not yet available on Fedora 34
+target-version = ['py38', 'py39']  # 'py310' is not yet available on Fedora 34
```

Not Fedora34 anymore.
Can add
```suggestion
target-version = ['py38', 'py39', 'py310', 'py311']
```

---

Thanks, I noticed that. I wanted to put this into separate PR to make PR and commit history clean.

---

As much as the pypy line could make sense to have in a separate PR, I think this one here is part of this PR.
You can amend + force push to have a clean history.

---

I still think that it should go into separate PR, as it requires bumping requirements in `pyproject.toml` file (`black==23.3.0`). It's better than both: PR and commits are atomic, it makes easier to reason about the changes in the future.

---

Okay, let's do that in another PR then!